scripts/scil_convert_sh_basis.py

```diff
@@ -41,32 +42,16 @@ def main():
     assert_inputs_exist(parser, args.input_sh)
     assert_outputs_exist(parser, args, args.output_name)
 
-    input_basis = args.sh_basis
-    output_basis = 'descoteaux07' if input_basis == 'tournier07' else 'tournier07'
-
-    sph_harm_basis_ori = sph_harm_lookup.get(input_basis)
-    sph_harm_basis_des = sph_harm_lookup.get(output_basis)
-
     sphere = get_sphere('repulsion724').subdivide(1)
     img = nib.load(args.input_sh)
-    data = img.get_data()
-    sh_order = find_order_from_nb_coeff(data)
-
-    b_ori, m_ori, n_ori = sph_harm_basis_ori(sh_order, sphere.theta,
-                                             sphere.phi)
-    b_des, m_des, n_des = sph_harm_basis_des(sh_order, sphere.theta,
-                                             sphere.phi)
-    l_des = -n_des * (n_des + 1)
-    inv_b_des = smooth_pinv(b_des, 0 * l_des)
-
-    indices = np.argwhere(np.any(data, axis=3))
-    for i, ind in enumerate(indices):
-        ind = tuple(ind)
-        sf_1 = np.dot(data[ind], b_ori.T)
-        data[ind] = np.dot(sf_1, inv_b_des.T)
-
-    img = nib.Nifti1Image(data, img.affine, img.header)
-    nib.save(nib.Nifti1Image(data, img.affine, img.header), args.output_name)
+    data = img.get_fdata(dtype=np.float32)
+
+    new_data = convert_sh_basis(data, sphere,
+                                sh_basis_type=args.sh_basis,
```

Comment:
Change this to input_basis, it crashs because this keyword argument does not exist